server/log_operation_manager.go

```diff
@@ -55,6 +56,7 @@ func createMetrics(mf monitoring.MetricFactory) {
 	resignations = mf.NewCounter("master_resignations", "Number of mastership resignations", logIDLabel)
 	isMaster = mf.NewGauge("is_master", "Whether this instance is master (0/1)", logIDLabel)
 	signingRuns = mf.NewCounter("signing_runs", "Number of times a signing run has succeeded", logIDLabel)
+	failedSigningRuns = mf.NewCounter("signing_runs", "Number of times a signing run has failed", logIDLabel)
```

Comment:
You forgot to update the name -- `failed_signing_runs` ?

Reply:
Looks like I did. Fixing it.